man/crypttab.xml

```diff
@@ -700,6 +700,28 @@
         order).</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>tpm2-measure-pcr=</option></term>
+
+        <listitem><para>Controls whether to measure the volume key of the encrypted volume to a TPM2 PCR. If
+        set to "no" (which is the default) no PCR extension is done. If set to "yes" the volume key is
+        measured into PCR 15. If set to a decimal integer in the range 0…23 the volume key is measured into
+        the specified PCR. The volume key is measured along with the activated volume name and its UUID. This
+        functionality is particularly useful for the encrypted volume backing the root file system, as it
+        then allows later TPM objects to be securely bound to root file system and hence installed
+        system.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><option>tpm2-measure-bank=</option></term>
+
+        <listitem><para>Selects the TPM2 PCR bank to measure the volume key into, as configured with
+        <option>tpm2-measure-pcr=</option> above. May be used multiple times to configure multiple PCR banks
+        to measure into. If not specified automatically determines available and used banks. Expects a
+        message digest name (e.g. <literal>sha1</literal>, <literal>sha256</literal>, …) as argument, to
+        identify the bank.</para></listitem>
```

Comment:
frankly, I don't expect anyone to use this. the automatic detection is what you want. this is in the territory of "let's make the common/default stuff easy, and the exotic stuff possible". This is exotic stuff...

comma separated is problematic because this is crypttab option sstring which is also comma separated.

anyway, I really wouldn't bother with this for now, it's a niche thing.

Reply:
(but anyway, I added this now)